conformance/utils/kubernetes/helpers.go

```diff
@@ -210,6 +210,34 @@ func WaitForGatewayAddress(t *testing.T, client client.Client, timeoutConfig con
 	return net.JoinHostPort(ipAddr, port), waitErr
 }
 
+// GatewayMustHaveZeroRoutes validates that the gateway has zero routes attached.  The status may indicate a single listener with not routes or no listeners.
+func GatewayMustHaveZeroRoutes(t *testing.T, client client.Client, timeoutConfig config.TimeoutConfig, gwName types.NamespacedName) {
+	var gotStatus *v1beta1.GatewayStatus
+	waitErr := wait.PollImmediate(1*time.Second, timeoutConfig.GatewayStatusMustHaveListeners, func() (bool, error) {
+		gw := &v1beta1.Gateway{}
+		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
For the sake of completeness, this timeout should likely also be configurable, we likely already have an existing timeout value that could work.

Reply:
done